scripts/showSOM.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#/usr/bin/python
```

Comment:
Wrong shebang; ! missing

Why changing to python2. I would prefer python3 for all scripts. Then also print_function import is not needed.